.github/workflows/macos.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         uses: mxschmitt/action-tmate@v3
 
       - name: Test
-        run: bazel test --spawn_strategy=local --copt="-g" --strip="never" --test_output=errors test:eventuals --test_arg=--gtest_shuffle --test_arg=--gtest_repeat=100
+        run: bazel test --spawn_strategy=local --copt="-g" --strip="never" --test_output=errors --experimental_ui_max_stdouterr_bytes=-1 test:eventuals --test_arg=--gtest_shuffle --test_arg=--gtest_repeat=100
```

Comment:
Can you add a comment explaining why this flag is here? (here and in the other yml files)